client/my-sites/themes/thanks-modal.jsx

```diff
@@ -107,13 +109,25 @@ class ThanksModal extends Component {
 		page( this.props.detailsUrl );
 	};
 
-	goToCustomizer = () => {
-		const { customizeUrl, shouldEditHomepageWithGutenberg } = this.props;
+	primaryAction = () => {
+		const {
+			activateGutenberg,
+			customizeUrl,
+			isUsingClassicEditor,
+			shouldEditHomepageWithGutenberg,
+			siteId,
+		} = this.props;
 
 		this.trackClick( 'thanks modal customize' );
 		this.onCloseModal();
 
-		shouldEditHomepageWithGutenberg ? page( customizeUrl ) : window.open( customizeUrl, '_blank' );
+		if ( isUsingClassicEditor && shouldEditHomepageWithGutenberg ) {
```

Comment:
`shouldEditHomepageWithGutenberg` is returning `isGutenbergTheme && isHomepageAPage` so in case that a user had post lists as front page they won't get the prompt.

I suggest using
```
import {  isThemeGutenbergFirst } from '../themes/selectors';
const isGutenbergTheme = isThemeGutenbergFirst( state, theme );
```